shop-ui/src/main/res/values/dimens.xml

```diff
@@ -52,9 +52,49 @@
 
     <dimen name="normal_1250">200dp</dimen>
 
-    <dimen name="dimen_item_category_text_size">18sp</dimen>
-    <dimen name="item_category_error_main_text_size">40sp</dimen>
-    <dimen name="item_category_error_secondary_text_size">35sp</dimen>
-    <dimen name="item_notification_main_text_size">24sp</dimen>
-    <dimen name="item_notification_secondary_text_size">14sp</dimen>
+    <dimen name="sp_normal_0">0sp</dimen>
+
+    <dimen name="sp_normal_6">1sp</dimen>
+    <dimen name="sp_normal_12">2sp</dimen>
+
+    <dimen name="sp_normal_25">4sp</dimen>
+    <dimen name="sp_normal_50">8sp</dimen>
+    <dimen name="sp_normal_60">10sp</dimen>
+    <dimen name="sp_normal_75">12sp</dimen>
+    <dimen name="sp_normal_100">16sp</dimen>
+
+    <dimen name="sp_normal_125">20sp</dimen>
+    <dimen name="sp_normal_150">24sp</dimen>
+    <dimen name="sp_normal_175">28sp</dimen>
+    <dimen name="sp_normal_200">32sp</dimen>
+
+    <dimen name="sp_normal_225">36sp</dimen>
+    <dimen name="sp_normal_250">40sp</dimen>
+    <dimen name="sp_normal_275">44sp</dimen>
```

Comment:
Im not sure that we will use sp large than 40. So please remove all sp dimens which large than 40sp

Reply:
fixed